### ISSUE TITLE
Training stage

### ISSUE THREAD

**Issue by zhangqiudan** [remega/OMCNN_2CLSTM#4]
Dear author,

Thank you very much for sharing your work, I am very interested in your work. I would be greatly appreciated if you can share your training demo, because i am not sure the training stages of OMCNN and 2c-lstm in the code. Thank you so much.

best regards.

**Comment by remega** [comment]
Thanks for your interest.  Actually, we haven't sorted out a clean version for training or fine-tunning.  I will make it depending on the future schedule. If you only need to know the training logic and detail, such as some hyper-parameters and train/loss function, I can provide a mess version., which can not be run directly. If so, pls contact me by email jianglai.china@aliyun.com

**Comment by zhangqiudan** [comment]
Thank you so much, I just sent you an email.
